scripts/data.js

```diff
@@ -0,0 +1,82 @@
+const movies = [
```

Comment:
```suggestion
export const movies = [
```
It's a good practice to export at the end of the file, but when it's only one variable/function it's alright to do it directly where you define it.